introspection-engine/connectors/sql-introspection-connector/Cargo.toml

```diff
@@ -31,6 +31,7 @@ user-facing-errors = {path = "../../../libs/user-facing-errors", features = ["sq
 [dependencies.quaint]
 features = ["single"]
 git = "https://github.com/prisma/quaint"
+branch = "xml_support"
```

Comment:
All occurrences of specific quaint branches will be reverted when the quaint branch has been merged.